### DISCUSSION TITLE
SensorWorker was restarted more than 10 times

### DISCUSSION THREAD

**Discussion by lmarchesi** [aaronwmorris/indi-allsky#1389] [General]
I have indi-allsky running on RPi5 and am using ASI676MC camera. No other sensors. When starting the service I get a pop-up message that SensorWorker was started more than ten times. 

In the log file (excerpt attached) I see many entries for 'sensor worker exception' which is preceded by indi.getTelescopeRaDec().

Is there something I've misconfigured possibly? I have never used indi for anything but indi-allsky. 

[indi-allsky-excerpt.log](https://github.com/user-attachments/files/16119782/indi-allsky-excerpt.log)

**Comment by aaronwmorris** [comment]
Hey, sorry about this.  I found this bug yesterday while working on other things.  The fix was part of #1387

**Comment by lmarchesi** [comment]
I did a pull after your fix for [#1388](https://github.com/aaronwmorris/indi-allsky/pull/1388 ) but I still see those errors in my log. Is the fix to suppress the pop-up and the errors I am seeing are 'normal'?

**Comment by lmarchesi** [comment]
I re-pulled from main and ran setup.sh and I am no longer seeing those messages. Sorry for the false alarm.

**Reply by aaronwmorris** [reply]
No worries.  Glad you got it worked out.

**Comment by rippleman** [comment]
I have a noob question...I have the same setup as lmarchesi and experiencing this issue.  If I was to pull from the main does it overwrite the configuration I've done (ie: exposure times, labels on images, etc.)?

**Reply by aaronwmorris** [reply]
@rippleman  The configuration is NOT overwritten when you upgrade.

**Reply by rippleman** [reply]
Perfect.  Thanks!
